examples/apps/quick_tap.py

```diff
@@ -0,0 +1,415 @@
+        self.assign_countdown_cube()
+
+    def blink_available_cubes(self):
+        '''Blinks the cubes which Cozmo did not select for himself.'''
+        for cube in self.cubes:
+            if cube.object_id != self.cozmo_player.cube.object_id:
+                cube.start_light_chaser()
+
+    def assign_countdown_cube(self):
+        '''Assigns the countdown cube to be whichever cube has not been selected by the player or Cozmo.'''
+        for cube in self.cubes:
+            if cube.object_id != self.cozmo_player.cube.object_id and cube.object_id != self.player.cube.object_id:
+                self.countdown_cube = cube
+                self.countdown_cube.stop_light_chaser()
+
+    def set_buzzer_lights(self):
+        '''Sets the buzzer cube lights based on the buzzer display type.'''
+        if self.buzzer_display_type == BUZZER_RED:
+            self.turn_on_buzzer_cubes_red()
+        elif self.buzzer_display_type == BUZZER_DIFFERENT:
+            self.turn_on_buzzer_cubes_different()
+        elif self.buzzer_display_type == BUZZER_SAME:
+            self.turn_on_buzzer_cubes_same()
+
+    def turn_on_buzzer_cubes_same(self):
+        '''Sets the buzzer cubes to the same randomly generated color pair.'''
+        same_colors = self.generate_random_buzzer_colors()
+        self.player.cube.set_light_corners(*same_colors)
+        self.cozmo_player.cube.set_light_corners(*same_colors)
+
+    def turn_on_buzzer_cubes_different(self):
+        '''Sets the buzzer cubes to different randomly generated color pairs.'''
+        player_cube_colors = self.generate_random_buzzer_colors()
+        cozmo_cube_colors = self.generate_random_buzzer_colors()
+        while player_cube_colors == cozmo_cube_colors:
+            cozmo_cube_colors = self.generate_random_buzzer_colors()
+        self.player.cube.set_light_corners(*player_cube_colors)
+        self.cozmo_player.cube.set_light_corners(*cozmo_cube_colors)
+
+    def turn_on_buzzer_cubes_red(self):
+        '''Sets the buzzer cubes to red.'''
+        self.player.cube.set_lights(cozmo.lights.red_light)
+        self.cozmo_player.cube.set_lights(cozmo.lights.red_light)
+
+    def generate_random_buzzer_colors(self):
+        '''Creates a list of different alternating colors, chosen randomly from LIGHT_COLORS_LIST.
+
+        Returns:
+            a list of Lights from LIGHT_COLORS_LIST'''
+        num_colors = len(LIGHT_COLORS_LIST)
+        x = random.randrange(num_colors)
+        y = random.randrange(num_colors)
+        while y == x:
+            y = random.randrange(num_colors)
+        return [LIGHT_COLORS_LIST[x], LIGHT_COLORS_LIST[y], LIGHT_COLORS_LIST[x], LIGHT_COLORS_LIST[y]]
+
+    def turn_off_buzzer_cubes(self):
+        '''Turns off both buzzer cubes' lights.'''
+        self.player.cube.set_lights_off()
+        self.cozmo_player.cube.set_lights_off()
+
+    def cubes_connected(self):
+        '''Returns true if Cozmo connects to all three cubes successfully.'''
```

Comment:
Should have a description of the method before the Returns line. Also Returns should be on its own line followed by a colon.